.github/workflows/ci.yml

```diff
@@ -0,0 +1,83 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+    - v2.0
+
+jobs:
+  scan_build:
+    runs-on: ubuntu-latest
+    container:
+      image: debian:buster
+    steps:
+      - name: install deps
+        run: |
+          apt-get update && apt-get install -yq build-essential autotools-dev lsb-release pkg-config automake autoconf libtool-bin clang-tools-7
+          apt-get install -yq cmake uuid-dev libssl-dev
+      - name: Checkout
+        uses: actions/checkout@v3
+      - run: pwd
+      - id: scan_build
+        run: /__w/libks/libks/scan_build.sh
```

Comment:
this feels horribly wrong, shouldn't we be able to do `./scan_build.sh`?

Reply:
fixed